src/Host.cpp

```diff
@@ -4164,3 +4160,20 @@ QPointer<TConsole> Host::parentTConsole(QObject* start) const
     }
     return qobject_cast<TConsole*>(ptr);
 }
+
+// The order in this is top, bottom, left, right
+void Host::setBorders(const std::tuple<int, int, int, int> borders)
+{
+    auto original = mBorders;
+    if (borders != original) {
+        mBorders = borders;
+        if (!mpConsole.isNull()) {
+            auto x = mpConsole->width();
+            auto y = mpConsole->height();
+            QSize s = QSize(x, y);
+            QResizeEvent event(s, s);
+            QApplication::sendEvent(mpConsole, &event);
+            mpConsole->raiseMudletSysWindowResizeEvent(x, y);
+        }
+    }
```

Comment:
Anti-pattern: Don't indent the whole function by two levels for practically no rewards.
```suggestion
    if (borders == original) {
        return;
    }
    mBorders = borders;
    if (mpConsole.isNull()) {
        return;
    }
    auto x = mpConsole->width();
    auto y = mpConsole->height();
    QSize s = QSize(x, y);
    QResizeEvent event(s, s);
    QApplication::sendEvent(mpConsole, &event);
    mpConsole->raiseMudletSysWindowResizeEvent(x, y);
```